packages/quantic/cypress/e2e/default-2/standalone-search-box/standalone-search-box-expectations.ts

```diff
@@ -17,9 +17,9 @@ function standaloneSearchBoxExpectations(
         .should(display ? 'exist' : 'not.exist')
         .logDetail(`${should(display)} display the input search box`);
     },
-    inputInitialized: (textarea = false) => {
+    searchBoxInputInitialized: () => {
```

Comment:
Why are we removing this? 
in our tests we still wanna confirm that the correct type of input is being used.